### PR TITLE
Adding Shopify to ABR Site

### PR DESCRIPTION
**Pull request by krisfrenette**

### CODE REVIEW

**Review by tbekolay — APPROVED** [review]
Made a few changes in e5ecc104a8ec765f82cdaec8b13b1b6f790c6e04; details in the commit message there. Take a look @krisfrenette and let me know if I missed something (Shopify's API is totally wack). Note that merging is blocked until we document and test the sales process.

File: js/shopify.js [hunk 1]
```diff
@@ -84,6 +97,8 @@ const generateComponent = ({
   });
 };
 
+var _scrollY; // Used on openModal and closeModal
```
Comment (krisfrenette) [inline js/shopify.js]: I don't think that you need `var` here. You should be able to use `let` still.

**Review by krisfrenette — COMMENTED** [review]
LGTM!

**Comment by tbekolay** [comment]
Task linked: [CU-q71kpg Backlog story](https://app.clickup.com/t/q71kpg)

**Pull request merged**